packages/broker/test/integration/plugins/operator/VoteOnSusppectNodeService.test.ts

```diff
@@ -0,0 +1,131 @@
+
+        await new Promise((resolve) => setTimeout(resolve, 5000)) // wait for events to be processed
+        const flaggerClient: MockProxy<StreamrClient> = mock<StreamrClient>()
+        const flaggerVoteService = new VoteOnSuspectNodeService(flaggerClient, flagger.operatorConfig)
+        await flaggerVoteService.start()
+
+        const targetClient: MockProxy<StreamrClient> = mock<StreamrClient>()
+        const targetVoteService = new VoteOnSuspectNodeService(targetClient, target.operatorConfig)
+        await targetVoteService.start()
+        expect(targetClient.subscribe.mock.calls.length).toBe(0)
+
+        const voterClient: MockProxy<StreamrClient> = mock<StreamrClient>()
+        const voterVoteService = new VoteOnSuspectNodeService(voterClient, voter.operatorConfig)
+        await voterVoteService.start()
+
+        logger.debug('deploying sponsorship contract')
+        const sponsorship = await deploySponsorship(config, adminWallet, {
+            streamId: streamId1 })
+        logger.debug('sponsoring sponsorship contract')
+        await (await token.connect(flagger.operatorWallet).approve(sponsorship.address, parseEther('500'))).wait()
+        await (await sponsorship.connect(flagger.operatorWallet).sponsor(parseEther('500'))).wait()
+
+        logger.debug('each operator delegates to its operactor contract')
+        logger.debug('delegating from flagger: ' + flagger.operatorWallet.address)
+        await (await token.connect(flagger.operatorWallet).transferAndCall(flagger.operatorContract.address,
+            parseEther('200'), flagger.operatorWallet.address)).wait()
+        logger.debug('delegating from target: ' + target.operatorWallet.address)
+        await (await token.connect(target.operatorWallet).transferAndCall(target.operatorContract.address,
+            parseEther('200'), target.operatorWallet.address)).wait()
+        logger.debug('delegating from voter: ' + voter.operatorWallet.address)
+        await (await token.connect(voter.operatorWallet).transferAndCall(voter.operatorContract.address,
+            parseEther('200'), voter.operatorWallet.address)).wait()
+        
+        logger.debug('staking to sponsorship contract from flagger and target and voter')
+        logger.debug('staking from flagger: ' + flagger.operatorContract.address)
+        await (await flagger.operatorContract.stake(sponsorship.address, parseEther('150'))).wait()
+        logger.debug('staking from target: ' + target.operatorContract.address)
+        await (await target.operatorContract.stake(sponsorship.address, parseEther('150'))).wait()
+        logger.debug('staking from voter: ' + voter.operatorContract.address)
+        await (await voter.operatorContract.stake(sponsorship.address, parseEther('150'))).wait()
+
+        logger.debug('registering node addresses')
+        await (await flagger.operatorContract.setNodeAddresses([await flagger.operatorContract.owner()])).wait()
```

Comment:
What's the purpose of setting the node addresses of the flagger here? Also why specifically `flagger.operatorContract.owner()`?